osversion/windowsbuilds.go

```diff
@@ -20,4 +20,8 @@ const (
 	// RS5 (version 1809, codename "Redstone 5") corresponds to Windows Server
 	// 2019 (ltsc2019), and Windows 10 (October 2018 Update).
 	RS5 = 17763
+
+	// RS6 (version 1903, codename "Redstone 6") corresponds to Windows Server
```

Comment:
I'm not sure if this is RS6 or just 19H1 🤔

Reply:
There is no RS6/Redstone 6, just 19H1. But I would prefer that we use the public version names for this instead of the code names, since the public names will be less confusing in the long run.

Maybe something like Win10_1903 for the identifier? Not sure how to make this idiomatic in Go (underscores seem rare), but I think we need to include the 10 to minimize confusion between a hypothetical future Windows 10 2003 and Windows 2003.

If we could go back to the others and update them as well, that would be good (can keep the existing RS5, etc. stuff for compatibility with existing callers).

Reply:
Thank you for the clarification. For the other suggestions we need consensus from other maintainers which should maybe done in a different PR. 
I‘ll leave this PR open for further feedback how to proceed.

Reply:
@StefanScherer

The official (RTM) name of Windows Server 18362 is Windows Server 1903. After RS5, there's no RS6 and the codename naming theme of RSx isn't used anymore. 

Since Windows Server 1903 has been released, I suggest to update the comment to use the official release name for 18362, the `Windows Server 1903`.
The codename (based on Windows Insiders) for 18362 is 19H1, and the next one is 20H1.

Reply:
The variety of names is definitely confusing 😓. Will the `YYHx` naming scheme apply both to Windows Client and Server? (My concern is that if we'll use, e.g. `WS1903` and there's a client version of using the same, that it may be equally confusing.

Note that renaming will affect some PR's I have open (https://github.com/moby/moby/pull/39100, https://github.com/docker/libnetwork/pull/2362, https://github.com/microsoft/hcsshim/pull/556) - not sure if the constants are already used elsewhere